Core/Config.h

```diff
@@ -144,7 +144,10 @@ struct Config {
 	int iBufFilter; // 1 = linear, 2 = nearest
 	bool bPartialStretch;
 	bool bStretchToDisplay;
-	bool bSmallDisplay;  // Useful on large tablets with touch controls to not overlap the image. Temporary setting - will be replaced by more comprehensive display size settings.
+	int iSmallDisplayZoom;  // Used to fit display into screen 0 = disabled, 1 = auto, 2 = x1, 3 = x2 etc..
+	float fSmallDisplayOffsetX;
+	float fSmallDisplayOffsetY;
+	float fSmallDisplayCustomZoom;
```

Comment:
Yeah, it can do zoom in and zoom out, I could word it differently maybe, fSmallDisplayCustomZoom is actually used for the zoom, so fractions are possible as well, iSmallDisplayZoom is only used inside the editor for integer zoom which also turns on manual edit.